components/cluster-coordinator/org.wso2.carbon.cluster.coordinator/org.wso2.carbon.cluster.coordinator.rdbms/src/test/java/org/wso2/carbon/cluster/coordinator/rdbms/test/RDBMSCoordinationStratergyBaseTest.java

```diff
@@ -72,12 +76,20 @@
     protected DataSource dataSource;
 
     protected void init() {
-        ConfigProvider configProvider = new ConfigProviderImpl(new YAMLBasedConfigFileReader("deployment.yaml"));
+        Path deploymentPath = null;
         try {
-            Map<String, Object> clusterConfig = configProvider.
-                    getConfigurationMap(CoordinationPropertyNames.CLUSTER_CONFIG_NS);
+            deploymentPath = Paths.get(ClassLoader.getSystemResource("conf/deployment.yaml").toURI());
```

Comment:
Will the resource path get resolved in windows because the path separator is '/' ?

Reply:
Will change it to use File.separator